release.sh

```diff
@@ -171,13 +171,18 @@ npm cache clean --force
 rm -rf node_modules/
 
 echo "==> Docker"
-docker build --no-cache --build-arg version=${VERSION} --tag gisaia/arlas-wui:${VERSION} .
+docker build -f docker/Dockerfile-production --no-cache --build-arg version=${VERSION} --tag gisaia/arlas-wui:${VERSION} .
+docker build -f docker/Dockerfile-production-no-analytics --no-cache --build-arg version=${VERSION} --tag gisaia/arlas-wui-no-analytics:${VERSION} .
 
 docker push gisaia/arlas-wui:${VERSION}
+docker push gisaia/arlas-wui-no-analytics:${VERSION}
 if [ "${STAGE}" == "stable" ] && [ "${IS_LATEST_VERSION}" == "YES" ];
     then
     docker tag gisaia/arlas-wui:${VERSION} gisaia/arlas-wui:latest
     docker push gisaia/arlas-wui:latest
+
+    docker tag gisaia/arlas-wui-no-analytics:${VERSION} gisaia/arlas-wui-no-analytics:latest
+    docker push gisaia/arlas-wui-no-analytics:latest
```

Comment:
Remove this part, `latest` is regular version of `arlas-wui`
Version with no-analytics is specific

Reply:
fixed